File: src/Chiffon.WebSite.FSharp/HomeController.fs
namespace Chiffon.WebSite.Controllers

open System.Web
open System.Web.Mvc

[<HandleError>]
type HomeController() =
    inherit Controller()
    member this.Index () =
        this.View() :> ActionResult
    member this.Contact() =
        this.View() :> ActionResult
    member this.About() =
        this.View() :> ActionResult

File: src/Chiffon/WebSite/Controllers/HomeController.cs
namespace Chiffon.WebSite.Controllers
{
    using System.Web.Mvc;

    public partial class HomeController : Controller
    {
        [HttpGet]
        public virtual ActionResult Index()
        {
            return View("~/Views/Home/Index.cshtml");
        }

        [HttpGet]
        public virtual ActionResult About()
        {
            return View("~/Views/Home/About.cshtml");
        }

        [HttpGet]
        public virtual ActionResult Contact()
        {
            return View("~/Views/Home/Contact.cshtml");
        }
    }
}
